kahuna/public/js/search/results.js

```diff
@@ -223,9 +223,7 @@ results.controller('SearchResultsCtrl', [
             $timeout(() => {
                 const latestTime = lastSearchFirstResultTime;
                 // Blank any 'until' parameter to look for new images
-                // TODO: if a manual until was set (e.g. using date
-                // picker), don't check for new images until now
-                search({since: latestTime, length: 0, until: null}).then(resp => {
+                search({since: latestTime, length: 0, until: $stateParams.until || null}).then(resp => {
```

Comment:
Could update the comment above to make this clearer (e.g. "Use explicit 'until' parameter, or else blank 'until'")